src/idl_gen_cpp.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Google Inc. All rights reserved.
```

Comment:
We can probably update these copyrights to 2023

Reply:
Will make the change once I'm on my personal laptop. On a side note, the CI is failing. But when I reproduced locally, flatc builds normally. Is there another way to reproduce this?

Reply:
I see. Thanks. TIL!